app/views/post_set_maintainers/index.html.erb

```diff
@@ -1,68 +1,21 @@
-      <th width="25%"></th>
+        <th>Set</th>
+        <th>Status</th>
+        <th>Date</th>
+        <th>Action</th>
       </thead>
       <tbody>
-      <% @invites.where(status: 'pending').each do |m| %>
+      <% @invites.where.not(status: "cooldown").each do |m| %>
         <% if m.post_set.is_public %>
           <tr>
             <td><%= link_to m.post_set.name, post_set_path(m.post_set) %></td>
             <td><%= m.status.capitalize %></td>
-            <td>
-              <%= link_to "Accept", approve_post_set_maintainer_path(m), 'data-confirm': 'Are you sure you want to accept this invite?' %>
-              <%= link_to "Ignore", deny_post_set_maintainer_path(m), 'data-confirm': 'Are you sure you want to ignore this invite?' %>
-              <%= link_to "Block", block_post_set_maintainer_path(m), 'data-confirm': 'Aer you sure you want to ignore and block future invites for this set?' %>
-            </td>
-          </tr>
-        <% end %>
-      <% end %>
-      </tbody>
-    </table>
-
-    <h2>Approved Invites</h2>
-    <table class="rounded nomargin" width="600px" style="margin-bottom:30px;">
-      <thead>
-      <th width="55%"><%= link_to "Set", action: "index", order: "set" %></th>
-      <th width="20%"><%= link_to "Status", action: "index", order: "status" %></th>
-      <th width="25%"></th>
-      </thead>
-      <tbody>
-      <% @invites.where(status: 'approved').each do |m| %>
-        <% if m.post_set.is_public %>
-          <tr>
-            <td><%= link_to m.post_set.name, post_set_path(m.post_set) %></td>
-            <td><%= m.status.capitalize %></td>
-            <td>
-              <%= link_to "Remove", deny_post_set_maintainer_path(m), 'data-confirm': 'Are you sure you want to remove yourself as maintainer of this set?' %>
-              <%= link_to "Block", block_post_set_maintainer_path(m), 'data-confirm': 'Aer you sure you want to remove yourself and block future invites for this set?' %>
-            </td>
-          </tr>
-        <% end %>
-      <% end %>
-      </tbody>
-    </table>
-
-    <h2>Blocked Invites</h2>
-    <table class="rounded nomargin" width="600px">
-      <thead>
-      <th width="55%"><%= link_to "Set", action: "index", order: "set" %></th>
-      <th width="20%"><%= link_to "Status", action: "index", order: "status" %></th>
-      <th width="25%"></th>
-      </thead>
-      <tbody>
-      <% @invites.where(status: 'blocked').each do |m| %>
-        <% if m.post_set.is_public %>
-          <tr>
-            <td><%= link_to m.post_set.name, post_set_path(m.post_set) %></td>
-            <td><%= m.status.capitalize %></td>
-            <td>
-              <%= link_to "Unblock", deny_post_set_maintainer_path(m), 'data-confirm': 'Are you sure you want to unblock invites for this set?' %>
-            </td>
+            <td><%= compact_time m.updated_at %></td>
+            <td><%= invite_links m %></td>
```

Comment:
Generally I dislike helpers that generate large amounts of html in rails because they are an easy way to make XSS injection mistakes through use of html_safe. This could be done using a partial and passing the invite member as a local to the view template. Something like `<%= render "post_set_maintainers/invite_links", invite: m %>`

If you're open to converting this, that's cool. If not, I'm open to accepting it as is too.

Reply:
No issue with changing that, thanks for the input (: